src/components/Theme/light-tokens.module.css

```diff
@@ -0,0 +1,109 @@
+/**
+ * Do not edit directly
+ * Generated on Mon, 22 Aug 2022 00:27:39 GMT
+ */
+
+:root {
+  --spectrum-gray-100: #f3f3f3;
+  --spectrum-gray-200: #e6e6e6;
+  --spectrum-gray-300: #b4b4b4;
+  --spectrum-gray-400: #9b9b9b;
+  --spectrum-gray-500: #818183;
+  --spectrum-gray-600: #68686a;
+  --spectrum-gray-700: #4f4f51;
+  --spectrum-gray-800: #363638;
+  --spectrum-gray-900: #1d1d1f;
+  --spectrum-gray-1000: #000000;
+  --spectrum-gray-000: #ffffff;
+  --spectrum-red-100: #ffe5e7;
+  --spectrum-red-200: #ffc2c6;
+  --spectrum-red-300: #ff9ea5;
+  --spectrum-red-400: #f96c73;
+  --spectrum-red-500: #f1465a;
+  --spectrum-red-600: #e32144;
+  --spectrum-red-700: #be1332;
+  --spectrum-red-800: #990620;
+  --spectrum-red-900: #650113;
+  --spectrum-red-1000: #320109;
+  --spectrum-red-000: #fff5f5;
+  --spectrum-blue-100: #c1ebf6;
+  --spectrum-blue-200: #96ddef;
+  --spectrum-blue-300: #6ad0e9;
+  --spectrum-blue-400: #3fc2e2;
+  --spectrum-blue-500: #14b4dc;
+  --spectrum-blue-600: #169bbc;
+  --spectrum-blue-700: #17829d;
+  --spectrum-blue-800: #18697e;
+  --spectrum-blue-900: #1a4f5e;
+  --spectrum-blue-1000: #1b363f;
+  --spectrum-blue-000: #ecf9fc;
+  --spectrum-green-100: #bbf1df;
+  --spectrum-green-200: #8ce7c9;
+  --spectrum-green-300: #5eddb4;
+  --spectrum-green-400: #2fd39e;
+  --spectrum-green-500: #00c988;
+  --spectrum-green-600: #00ad76;
+  --spectrum-green-700: #007552;
+  --spectrum-green-800: #005940;
+  --spectrum-green-900: #003d2e;
+  --spectrum-green-1000: #00291f;
+  --spectrum-green-000: #f3fcf9;
+  --spectrum-yellow-100: #ffeebe;
+  --spectrum-yellow-200: #ffe397;
+  --spectrum-yellow-300: #ffd971;
+  --spectrum-yellow-400: #ffce4a;
+  --spectrum-yellow-500: #ffc424;
+  --spectrum-yellow-600: #ffad1f;
+  --spectrum-yellow-700: #fa8900;
+  --spectrum-yellow-800: #e55c00;
+  --spectrum-yellow-900: #c73f00;
+  --spectrum-yellow-1000: #752100;
+  --spectrum-yellow-000: #fff8e4;
+  --global-fontFamilies-ivypresto-display: IvyPresto Display;
+  --global-fontFamilies-sf-compact: 'SF Compact', system-ui, sans-serif;
```

Comment:
Now I'm just adding the back up fonts to SF Compact, we can revisit this later and add backups for the other fonts if necessary. cc @jaredshaunsmith